test/clj_sqs_extended/core_test.clj

```diff
@@ -33,11 +24,11 @@
         (let [message (sqs-ext/receive-message client url {:format format})]
           (is (= test-message (:body message))))))))
 
-(deftest can-auto-delete-message
+(deftest can-auto-delete-messageX
```

Comment:
what's `messageX`?

Reply:
Damn. That happens when you slip while using your run-tests shortcut :sob: